### DISCUSSION TITLE
TYPO in Written Lesson

### DISCUSSION THREAD

**Discussion by MostPow3rful** [Cyfrin/foundry-full-course-cu#2251] [General]
GM

[Foundry FUnda](https://updraft.cyfrin.io/courses/foundry/foundry-fund-me/refactoring-testing?lesson_format=transcript)
![image](https://github.com/user-attachments/assets/bdffbb78-e66c-45f9-a92e-bed2a287dc18)
Must Change to `AggregatorV3Interface`

GN

**Comment by EngrPips** [comment]
Thanks for pointing this out @MostPow3rful, But I think using the issues tab is more appropriate for pointing errors in codebase

**Comment by AztecWarlord** [comment]
Create a pull request to update the code.

**Comment by MostPow3rful** [comment]
GM All
OK

**Reply by EngrPips** [reply]
Gm
